### DISCUSSION TITLE
NCIDS Documentation Site - Design Principles Page Content

### DISCUSSION THREAD

**Discussion by andyvanavery31** [NCIOCPL/ncids#1417] [NCIDS Launch]
Per the Batch 2 review of the NCIDS Pilot from Jorge, his response on the review was, "Is this content going to be expanded? I don’t think it’s enough content to warrant it’s own page or main nav menu. Obviously keep it if content will be expanded with examples or visual aids in the future. It’s mentioned above that this page is not content heavy because it links and directs users (but doesn’t seem like that functionality is there for the pilot?)"

This discussion is to make a determination on if a change will be made to this page, and if a change will be made, what those changes will be. Any changes will be made to improve the site in preparation for the NCIDS For All Launch.

**Comment by andyvanavery31** [comment]
@adrianacastaneda investigating to determine if changes should be made to this page. Will bring findings to the Requirements meeting on 13DEC2023 to discuss with the team.

**Comment by sarinapadilla** [comment]
We've got #1598 and #1599 handling this!
